tests/test_update_api_key.py

```
@@ -366,6 +342,77 @@ def test_update_api_key_skips_blank_env_lines(self, tmp_path, fake_crypto):
+
+        with patch("update_api_key.Path.home", return_value=tmp_path):
+            update_api_key.update_api_key(free_key, profile="free")
+
+        lines = env_file.read_text().splitlines()
+        assert any(line.startswith("OPENAI_API_KEY=encrypted:premium-value") for line in lines)
+        assert any(line.startswith("OPENAI_API_KEY_FREE=") for line in lines)
+
+        meta = json.loads((tmp_path / ".cursor" / "key.meta.json").read_text())
+        assert "free" in meta["profiles"]
+
+    def test_rotate_api_key_creates_backups(self, tmp_path, fake_crypto):
```

Ruff (0.13.2)

Unused method argument: `fake_crypto`

(ARG002)